gecoscc/tasks.py

```diff
@@ -15,6 +15,7 @@
 import os
 import subprocess
 import traceback
+import pymongo
```

Comment:
Instead of import pymongo and do raw querys here, move the methods to gecoscc.rules and import from there